zest/releaser/utils.py

```diff
@@ -8,12 +8,14 @@
 import subprocess
 import sys
 import textwrap
+
 try:
     from tokenize import detect_encoding
 except ImportError:
     detect_encoding = None
 try:
     import chardet
+
```

Comment:
This newline looks strange to me.
Does pep8 want it?

Reply:
I think I ran `autopep8` over it and maybe that did it. But when I remove the line and run it again, it does not add it.
Looks strange indeed. I will remove it.
I could do that after merge too, maybe with some automated import sorting with `isort`.

Reply:
O well, I will do it on this PR. I need to resolve a conflict anyway before this can be merged.